extensions/wikia/AdEngine/js/pageFairDetection.js

```diff
@@ -60,6 +71,7 @@ define('ext.wikia.adEngine.pageFairDetection', [
 	}
 
 	return {
+		canBeInitialized: canBeInitialized,
```

Comment:
Did you expose this method basically for Mercury?

1. I am curious why not use adContext directly on mercury?
2. Do we have the same issue on mercury for SourcePoint?
```